README.md

```diff
@@ -48,4 +49,3 @@ See the [contribution wiki page](https://github.com/Akshay0701/food_delivery_app
 License
 -------
 [GPL-3.0 License](https://github.com/ankidroid/Anki-Android/blob/master/COPYING)  
-
```

Comment:
is there any reason for removing this line?

Reply:
actually this line is not necessary in the documentation at end, so i removed it